assets/src/edit-story/components/thumbnail/styles.js

```diff
@@ -107,14 +107,6 @@ export const Background = styled.div(
     display: flex;
     border-radius: ${theme.borders.radius.small};
 
-    img,
```

Comment:
😿 
<img width="278" alt="Screen Shot 2021-07-15 at 11 08 56 AM" src="https://user-images.githubusercontent.com/10720454/125836545-cc93c03f-48ae-447c-8d42-bea2e30d09c3.png">

Reply:
maybe let's add some more specific selector here that applies these styles to images/videos rendered, but doesn't reach in and effect images/videos within a rendered story

Reply:
ah good catch - I'll fix that :)

Reply:
Should be fixed now!

<img src="https://user-images.githubusercontent.com/22185279/125858200-77abbd8c-5323-48d3-a8de-40304fbe9c53.png" height="350px" />